src/shadowbox/infrastructure/get_port.spec.ts

```diff
@@ -33,8 +33,8 @@ describe('PortProvider', () => {
     it('returns free port', async () => {
       const ports = new get_port.PortProvider();
       const server = await listen();
-      expect(await ports.reserveFirstFreePort(server.address().port))
-          .toBeGreaterThan(server.address().port);
+      const initialPort = (server.address() as net.AddressInfo).port;
+      expect(await ports.reserveFirstFreePort(initialPort).toBeGreaterThan(initialPort);
```

Comment:
Oops, missing parenthesis:
```suggestion
      expect((await ports.reserveFirstFreePort(initialPort)).toBeGreaterThan(initialPort);
```